### DISCUSSION TITLE
SoundStream overfitting

### DISCUSSION THREAD

**Discussion by Dannynis** [lucidrains/audiolm-pytorch#121] [Q&A]
Hey, I'm trying to overfit the SoundStream model on 10 examples using the most basic setup:

no attentions
only reconstruction loss
https://colab.research.google.com/drive/1pEA1vQwxNnrj7g6B_zOi4laEc82VMFB8?usp=sharing
But still couldn't managed to make it work....
Any suggestions on why is it happening ?
(p.s disabling the discriminator causes error at the trainer - .item() should be discarded)
Thank you !

**Comment by lucidrains** [comment]
@Dannynis oh oops, disabling the discriminators should work now

**Reply by Dannynis** [reply]
Thank you, much appreciated :)

**Comment by amitaie** [comment]
hey @Dannynis  I have a follow-up question:
 I'm trying to train SoundStream only with reconstruction loss (for debugging) and it stopped going down after a while (around 900, when using norm stft). did it happen to you to? any thoughts about it?
